### PR TITLE
Database week2

### PR DESCRIPTION
**Pull request by AliRezaHassanloo2731**

### CODE REVIEW

File: databases/week2/homework.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    updated, due_date, status_id, user_id);
+--Change the title of a task
+update mydb.task set title='do homework' where id=7;
+
+--Change a task due date
+update mydb.task set due_date='2024-07-10 20:16:15' where id =3;
+--Change a task status
+update mydb.task set status_id='1' where title='do homework';
+--Mark a task as complete
+UPDATE mydb.task set status_id='3' WHERE title='do homework';
```
Comment (NatsGt) [inline databases/week2/homework.sql:13]: Try to not hardcode the status_id but look for the id where status = 'done'

File: databases/week2/homework.sql [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+--Create an index on the name column of the student table.
+CREATE INDEX index_name ON student (name);
+--Add a new column to the class table named status
+-- which can only have the following 
+--values: not-started, ongoing, finished (hint: enumerations).
+ALTER TABLE student ADD COLUMN status 
+ENUM('not-started','ongoing','finished')
+
+--Part 3: More queries
```
Comment (NatsGt) [inline databases/week2/homework.sql:54]: The queries look good, but I think you needed to use other tables and values, look at the instructions for this part.

File: databases/week2/homework.sql [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT * FROM task INNER JOIN user ON email LIKE '%spotify.com';
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+
+SELECT * FROM task INNER join user on `user`.name ='Donald Duck' AND status_id='1' ;
```
Comment (NatsGt) [inline databases/week2/homework.sql:59]: Dont hardcode the status_id, instead do a join on status table.

File: databases/week2/paradim.png
Comment (NatsGt) [inline databases/week2/paradim.png:1]: I think you need to rearrange this diagram. My guess is that you want to do a db where a CUSTOMER can ORDER a PRODUCT. In this case, there should be a table for orders connecting customer and product. Tell me if you have another idea for this db.

**Review by NatsGt — CHANGES_REQUESTED** [review]

File: databases/week2/paradim.png
Comment (AliRezaHassanloo2731) [reply databases/week2/paradim.png:1]: exactly, overall i want design sperso hose which the core and focus on customer,i will try again

File: databases/week2/homework.sql (hunk 1 above)
Comment (AliRezaHassanloo2731) [reply databases/week2/homework.sql:13]: select title, name from task inner join status on 
task.status_id=status.id where name='Done';
is this your mean?

Comment (NatsGt) [reply databases/week2/homework.sql:13]: dont need a join here, just a simple query on the status table to get the id of status 'done', and then put that query where you put in inside your update. `UPDATE task set status_id = (Select ...`

**Pull request closed without merging**